### PR TITLE
Add Bukashk0zzz FilterBundle

### PR DESCRIPTION
**Pull request by Bukashk0zzz**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: bukashk0zzz/filter-bundle/2.0/config/packages/bukashk0zzz_filter.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bukashk0zzz_filter:
+    # Enable if you need auto filtering form data before constrain(Validation) check
```
Comment (stof) [inline bukashk0zzz/filter-bundle/2.0/config/packages/bukashk0zzz_filter.yaml]: isn't there a typo here ?

Comment (Bukashk0zzz) [reply bukashk0zzz/filter-bundle/2.0/config/packages/bukashk0zzz_filter.yaml]: Checked twice. I think no. All correct.

Comment (fabpot) [reply bukashk0zzz/filter-bundle/2.0/config/packages/bukashk0zzz_filter.yaml]: Shouldn't it be `constraint` instead of `constrain`?

Comment (Bukashk0zzz) [reply bukashk0zzz/filter-bundle/2.0/config/packages/bukashk0zzz_filter.yaml]: My bad. Updated.

**Pull request merged**